### ISSUE TITLE
[Accessibility]: Action items bar keybinding not discoverable to screen readers

### ISSUE THREAD

**Issue by tjkandala** [sourcegraph/sourcegraph#36854]
### Audit type

Screen reader navigation

### User journey audit issue

#34056

### Problem description

When on a file page and focusing the "open/close extensions panel" button, it is not apparent how one should enter the action items bar/extensions panel.

### Expected behavior

The screen reader may announce the keyboard shortcut (down arrow) to enter the action items bar/extensions panel. Within the action items bar, we may use the [toolbar role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/toolbar_role).

### Additional details

![Screen Shot 2022-06-08 at 3 58 43 PM](https://user-images.githubusercontent.com/37420160/172706022-13d98d33-a735-4de5-b3e7-bbebad39a0d7.png)

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @muratsu @jjinnii @ryankscott - the "team/integrations" label was applied to this issue.
